adam/curriculum/phase1_curriculum.py

```diff
@@ -705,7 +709,7 @@ def falling_template(
                             theme,
                             SpatialPath(
                                 operator=TOWARD,
-                                reference_object=ground,
+                                reference_source_object=ground,
```

Comment:
If the source is the beginning of the path, for falling wouldn't we expect the path to begin away from the ground and end on the ground, so the `reference_source_object` would be whatever it fell off of?

Reply:
@paynesa Yes, I agree. That's the confusion that led to wanting to be able to distinguish between the two. So the question is when we have a spatial path do we: Require at least one of a source or destination or Require both?

For "Falling" I think the source should be `Region(ground, distance=PROXIMAL or DISTAL, direction=Gravitational_up)` and the destination is `Region(ground, distance=EXTERIOR_BUT_IN_CONTACT_WITH, direction=Gravitational_up)` right?

Adding this level of distinction definitely helps resolve ambiguity but means some thought needs to be put into how we apply it to the curriculum (in both our dictionaries and the templates) so we're consistent across all the actions.

Reply:
@LichMaster98 I agree with your semantics for falling if there's not a specified origin (otherwise I think we'd want to start in contact with that origin?). 

I think requiring both can make sense -- for ones where there's not a clear direction (e.g. when we're just introducing a path for the purpose of imprecise temporal descriptions), we can probably just make the source and the origin the same since there's no motion with regards to that path, but that does seem a bit hackish. Happy to discuss this more!

Also small sidenote: I think you want to have this PR not close #932 since source/destination isn't fully implemented yet :)

Reply:
Ah! My intent is to add to this PR with the resolution of the source and destinations after we discuss how we want to resolve them. But figured getting a review of the technical changes first would help in aiding the discussion for how to go about modifying the templates.